digitransit-component/packages/digitransit-component-icon/src/index.js

```diff
@@ -44,17 +44,17 @@ import ModeBusLocal from './assets/bus-local.svg';
 import ModeRail from './assets/mode_rail.svg';
 import ModeTram from './assets/mode_tram.svg';
 import ModeFerry from './assets/mode_ferry.svg';
-import ModeCitybike from './assets/mode_citybike.svg';
+import ModeVehicleRentalStation from './assets/mode_citybike.svg';
```

Comment:
Actually I think in this file we should call these SVG imports BikeRentalStation instead of VehicleRentalStation because the icons actually have bikes in them.